module/data/item/race.mjs

```diff
@@ -113,25 +113,6 @@ export default class RaceData extends SystemDataModel.mixin(ItemDescriptionTempl
   _onCreate(data, options, userId) {
     if ( (game.user.id !== userId) || this.parent.actor?.type !== "character" ) return;
     const updates = { "system.details.race": this.parent.id };
-    const attributes = this.parent.actor.system.attributes;
-
-    for ( const key of Object.keys(CONFIG.DND5E.movementTypes) ) {
-      if ( this.movement[key] > attributes.movement[key] ) {
-        updates[`system.attributes.movement.${key}`] = this.movement[key];
-      }
-    }
-    if ( this.movement.hover ) updates["system.attributes.movement.hover"] = true;
-    updates["system.attributes.movement.units"] = this.movement.units;
-
-    for ( const key of Object.keys(CONFIG.DND5E.senses) ) {
-      if ( this.senses[key] > attributes.senses[key] ) {
-        updates[`system.attributes.senses.${key}`] = this.senses[key];
-      }
-    }
-    if ( this.senses.special ) {
-      updates[system.attributes.senses.special] = [attributes.senses.special, this.senses.special].filterJoin(";");
-    }
-    updates["system.attributes.senses.units"] = this.senses.units;
 
     if ( this.type.value ) updates["system.details.type.value"] = this.type.value;
```

Comment:
Should this be derived from the race item as well? We have existing movement and senses editors that we probably don't want to remove, but creature type on characters is new, so there's nothing to take away there. Perhaps it should not be editable at all, nor exist on the character data model, and should come purely from the race item? Anyone wanting to modify their creature type should do it via the race item, in the same way that editing level is done via the class item.